styles.css

```diff
@@ -1,31 +1,327 @@
+/*    4.3 Section Two
+/*    4.4 Section Three
+      4.5 Section Three
+/*    4.6 Footer */
+/*5. Media Queries*/
 
+
+/* ------------------------------- 1. Global Styles -------------------------------- */
+body {
+    width: 100%;
+    overflow-x: hidden;
+}
 /* ------------------------------- 2. Typography ----------------------------------- */
 
+
+body {
+    font-family: "Roboto", sans-serif;
+    font-weight: 300;
+    font-style: normal;
+    font-size: 16px;
+}
+
+.titel {
+
+    font-family: "Merriweather", serif;
+    font-weight: 500;
+    font-style: normal;
+    font-size: 40px;
+}
+
 /* ------------------------------- 3. Layout --------------------------------------- */
 
-/* --------------- 3.1 Reusables ----------------------- */
+article {
+    display: flex;
+    flex-direction: column-reverse;
+    justify-content: space-between;
+
+}
+
+.sectionTwo {
+    background-color: #F2F0FF;
+    display: flex;
+    flex-direction: column;
+}
+
+.blocks {
+    color: #4845E4;
+    font-weight: 400;
+    font-size: 18px;
+    font-family: "Merriweather", serif;
+    padding: 0 0 10px 0;
+}
+
+.work {
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+    font-weight: 450;
+    font-size: 18px;
+    gap: 15px;
+    flex-wrap: wrap;
+}
 
+/* --------------- 3.1 Reusables ----------------------- */
 /* --------------- 3.2 Other --------------------------- */
 
-/* ------------------------------- 4. Areas ---------------------------------------- */
+.squiggleLineStripe {
+    position: relative;
+    top: 100%;
+    left: 47%;
+}
+
+img {
+    flex-grow: 0;
+    flex-shrink: 1;
+    padding: 50px 0 0 0;
+}
 
+.image-wrapper {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    flex-basis: 300px;
+    flex-grow: 0;
+    flex-shrink: 1;
+}
+
+/* ------------------------------- 4. Areas ---------------------------------------- */
 /* --------------- 4.1 Header ------------------------- */
+header {
+    position: relative;
+    width: 100%;
+    background: rgb(72,69,228);
+    background: linear-gradient(90deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 55%);
+    color: #F2F0FF;
+}
+
+nav {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    gap: 50px;
+    list-style-type: none;
+    color: white;
+}
+a {
+    list-style-type: none;
+    color: white;
+    text-decoration: none;
+}
+.headerText {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 20px;
+    margin: 150px 0 200px 0;
+    flex-wrap: wrap;
+    text-align: center;
+}
+
+h2 {
+    font-size: 36px;
+}
+
+.LetsWorkTogether {
+    font-family: "Roboto", sans-serif;
+    font-weight: 300;
+    font-style: normal;
+    text-transform: uppercase;
+    font-size: 10px;
+    border-radius: 15px;
+    padding: 10px 20px;
+    background-color: #ED4385;
+    color: #F2F0FF;
+    border: none;
+    box-shadow: 6px 10px 10px rgba(0, 0, 0, 0.3);
+}
+
+.send {
```

Comment:
Bij een button is het vaak gebruiksvriendelijk om het css property cursor toe te voegen.